CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 Changes that are currently in development and have not been released yet.
 
+
+## [0.13.3](https://github.com/cossacklabs/themis/releases/tag/0.13.2), October 12th 2020
+
+**Hotfix for Themis CocoaPods and Xcode12:**
+
+- Default Themis podspec is using OpenSSL 1.0.2u again ("themis/themis-openssl"). OpenSSL 1.1.1g podspec ("themis/openssl") might be broken for Xcode12, fixing is in progress. BoringSSL podspec ("themis/themis-boringssl") is available too.
```

Comment:
> BoringSSL podspec ("themis/themis-boringssl") is available too.

But *does it work*? (I assume so, since it compiles BoringSSL from source on the developer machine when building application, so that should invoke the necessary Xcode magic.)

Reply:
well, tests pass

Reply:
> well, tests pass

They did pass for 0.13.2 too and yet it turned out to be broken after all :shrug: